drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/java/org/drools/workbench/screens/drltext/backend/server/DRLTextEditorServiceImplCDITest.java

```diff
@@ -175,6 +176,27 @@ public void testNumericalTypes() throws Exception {
         Assertions.assertThat(validationMessages).isEmpty();
     }
 
+    @Test
+    public void testInvalidContentDRL() throws Exception {
+        final String error102 = "[ERR 102] Line 1:7 mismatched input ''";
+        final String error107 = "[ERR 107] Line 1:0 mismatched input 'asd' " +
+                "expecting one of the following tokens: " +
+                "'[package, unit, import, global, declare, function, rule, query]'.";
+        final String errorPackage = "Parser returned a null Package";
+
+        //This regexp is looking for kbase instance names like "[KBase: kbase1]" in validation messages
+        final String regexpExpression = "\\A\\[(.*?)\\][:]\\s";
+
+        validateResource(INVALID_CONTENT_DRL);
+        Assertions.assertThat(validationMessages).hasSize(6);
+        Assertions.assertThat(validationMessages
+                                      .stream()
+                                      .map(validationMessage -> validationMessage
+                                              .getText()
+                                              .replaceAll(regexpExpression, "")))
+                .contains(error102, error107, errorPackage);
```

Comment:
Please use `containsOnly` instead of `contains` otherwise looks good.

Reply:
thnx, done